ocfweb/account/vhost.py

```diff
@@ -216,18 +224,41 @@ class VirtualHostForm(Form):
                this box and move on.)',
     )
 
+    # required disclaimer by the university hostmaster
+    website_hostmaster_policy = forms.BooleanField(
+        label='You acknowledge that all relevant university \
+               policies will be followed, including those pertaining \
+               to <a href="https://dac.berkeley.edu/web-accessibility">\
+               campus website accessibility</a>',
+    )
+
+    # also see __init__
+    your_position = forms.CharField(
+        min_length=1,
+        max_length=64,
+    )
+
     your_email = forms.EmailField(
         label='Your email address:',
         min_length=1,
         max_length=64,
     )
 
-    # also see __init__
-    your_position = forms.CharField(
+    university_contact = forms.EmailField(
+        label='Contact email address for the university (may be the same as your email):',
```

Comment:
Hmmm not sure. We do ask for an email and that doesn't have a default value, but this is a "permanent contact" type of thing.

Reply:
Theoretically for as long as a group's site is hosted with us, our email-on-file should also be an up-to-date contact, but we know how bad people are with that...

Reply:
I'm going to stick on the side of no default here, since we want people to give an address they'll read (not one that they use for OCF group account setup). The university might also get suspicious of a bunch of @ocf.berkeley.edu emails being contacts, even if they are forwarded.